src/contexts/v3/projectContext.ts

```diff
@@ -0,0 +1 @@
+export { V2ProjectContext as V3ProjectContext } from '../v2/projectContext'
```

Comment:
Reuse V2 context in V3. If it turns out we need a custom V3 context, all we need to do is remove this line, define a `V3ProjectContext` const and export that instead.